pypfopt/plotting.py

```diff
@@ -17,7 +17,7 @@
 from . import CLA, EfficientFrontier, exceptions, risk_models
 
 try:
-    plt.style.use("seaborn-deep")
+    plt.style.use("seaborn-v0_8-deep")
```

Comment:
most likely better to use "default" instead of "seaborne-*" for backwards compatibility with matplotlib < 3.8.0